### ISSUE TITLE
Rollup 0.35.4 breaks my build

### ISSUE THREAD

**Issue by alangpierce** [rollup/rollup#919]
I'm getting a crash with this stack trace:
```
Cannot read property 'initialise' of null
TypeError: Cannot read property 'initialise' of null
    at ForStatement.initialise (/home/travis/build/alangpierce/bulk-decaffeinate/node_modules/rollup/src/ast/nodes/ForStatement.js:24:14)
    at BlockStatement.initialise (/home/travis/build/alangpierce/bulk-decaffeinate/node_modules/rollup/src/ast/nodes/BlockStatement.js:43:9)
    at eachChild.child (/home/travis/build/alangpierce/bulk-decaffeinate/node_modules/rollup/src/ast/Node.js:64:34)
    at IfStatement.eachChild (/home/travis/build/alangpierce/bulk-decaffeinate/node_modules/rollup/src/ast/Node.js:21:6)
    at IfStatement.initialise (/home/travis/build/alangpierce/bulk-decaffeinate/node_modules/rollup/src/ast/Node.js:64:8)
    at IfStatement.initialise (/home/travis/build/alangpierce/bulk-decaffeinate/node_modules/rollup/src/ast/nodes/IfStatement.js:11:11)
    at eachChild.child (/home/travis/build/alangpierce/bulk-decaffeinate/node_modules/rollup/src/ast/Node.js:64:34)
    at IfStatement.eachChild (/home/travis/build/alangpierce/bulk-decaffeinate/node_modules/rollup/src/ast/Node.js:21:6)
    at IfStatement.initialise (/home/travis/build/alangpierce/bulk-decaffeinate/node_modules/rollup/src/ast/Node.js:64:8)
    at IfStatement.initialise (/home/travis/build/alangpierce/bulk-decaffeinate/node_modules/rollup/src/ast/nodes/IfStatement.js:11:11)
```

See these greenkeeper PRs for links to the failing CI builds (which I also reproduced locally):
https://github.com/alangpierce/bulk-decaffeinate/pull/23
https://github.com/decaffeinate/decaffeinate/pull/415

Looks like the crash was probably caused by this commit: https://github.com/rollup/rollup/commit/9d9cfccd32f85bd19dc53c5c0164d120687f2db6

**Comment by Rich-Harris** [comment]
Thanks – if my understanding of the bug is correct, it should be fixed by 0.35.5

**Comment by alangpierce** [comment]
Yep, I just tried the build again with 0.35.5 and it works now. Thanks!
